test/functionalTest/cases/2991_invalid_description_query_error_response/invalid_description_query_error_response.test

```diff
@@ -0,0 +1,70 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
+#
+# This file is part of Orion Context Broker.
+#
+# Orion Context Broker is free software: you can redistribute it and/or
+# modify it under the terms of the GNU Affero General Public License as
+# published by the Free Software Foundation, either version 3 of the
+# License, or (at your option) any later version.
+#
+# Orion Context Broker is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU Affero
+# General Public License for more details.
+#
+# You should have received a copy of the GNU Affero General Public License
+# along with Orion Context Broker. If not, see http://www.gnu.org/licenses/.
+#
+# For those usages not covered by this license please contact with
+# iot_support at tid dot es
+
+# VALGRIND_READY - to mark the test ready for valgrindTestSuite.sh
+
+--NAME--
+Invalid description in POST /op/query error response
+
+--SHELL-INIT--
+dbInit CB
+brokerStart CB
+
+--SHELL--
+
+#
+# 01. Attempt to perform a batch query with invalid metadata field, and see it fail with a proper error description
+#
+
+echo "01. Attempt to perform a batch query with invalid metadata field, and see it fail with a proper error description"
+echo "============================================================================================="
```

Comment:
Minor: as many "=" as chars in the echo message previous to this line.

Reply:
Fixed in 9f260d5